### PR TITLE
Make getUserSettings to be called on root component and fix loading of Settings

### LINKED ISSUE #2084: Make getUserSettings to be called on root component and fix loading of Settings
**Actual Behaviour**
Currently the showAdminService and listusersettings are mounted when static app bar is loaded. 

**Expected Behaviour**
the showAdminService and listusersettings should mounted on root element.

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by harsh-jindal**
Fixes #2084 

Changes: Remove the code which was causing the settings to be not functional.
Redux is already integrated in Static App bar. 
Actions.getSettings() functions in index.js calls the getUserSettings function which lists the user settings.
Demo Link: https://pr-2085-fossasia-susi-web-chat.surge.sh

### CODE REVIEW

**Comment by harsh-jindal** [comment]
@fragm3 @akshatnitd Please review.
I believe you have made the same changes in this PR https://github.com/fossasia/chat.susi.ai/pull/2078 to fix the loading. But I think it would be better to merge this PR as then I can work on https://github.com/fossasia/chat.susi.ai/issues/2083 issue and it might take time to resolve problems in  https://github.com/fossasia/chat.susi.ai/pull/2078

**Comment by fragm3** [comment]
@harsh-jindal I am working on the issue of Travis error at the moment. I had branched out #2079 from #2078. If this gets merged, I'll get conflicts in both created PR.

**Comment by harsh-jindal** [comment]
@akshatnitd I have modified the PR as before the Admin tab was not seen on settings and about page.
@fragm3 I guess you won't get conflicts as we both have made the same changes.

**Comment by harsh-jindal** [comment]
@akshatnitd Made the changes. Please review.

File: src/App.js [hunk 1]
```diff
@@ -63,6 +64,7 @@ class App extends Component {
 
     actions.getApiKeys();
     accessToken && actions.getAdmin();
+    accessToken && getSettings();
```
Comment (akshatnitd) [inline src/App.js]: Do we not set the username in the cookies in the then block? Check CMS

**Comment by harsh-jindal** [comment]
@fragm3 Cool, I'll close this PR then

**Pull request closed without merging**